opencti-platform/opencti-front/src/private/components/techniques/data_components/DataComponentLineDummy.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import makeStyles from '@mui/styles/makeStyles';
+import ListItem from '@mui/material/ListItem';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import ListItemText from '@mui/material/ListItemText';
+import { KeyboardArrowRight } from '@mui/icons-material';
+import Skeleton from '@mui/material/Skeleton';
+import { Theme } from '../../../../components/Theme';
+import { DataColumns } from '../../../../components/list_lines';
+
+const useStyles = makeStyles<Theme>((theme) => ({
+  item: {
+    paddingLeft: 10,
+    height: 50,
+  },
+  itemIcon: {
+    color: theme.palette.primary.main,
+  },
+  bodyItem: {
+    height: 20,
+    fontSize: 13,
+    float: 'left',
+    whiteSpace: 'nowrap',
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+    paddingRight: 5,
+  },
+  goIcon: {
+    position: 'absolute',
+    right: -10,
+  },
+}));
+
+const DataComponentLineDummy = ({ dataColumns }: { dataColumns?: DataColumns }) => {
```

Comment:
Why is dataColumns not mandatory ?

Reply:
Because we can call it without data colums to have just one shadow line
But maybe it's better to have two different components to do that ?

Reply:
Fix after discussion